examples/android/build.gradle

```diff
@@ -1,5 +1,7 @@
-apply plugin: 'com.android.application'
-apply plugin: 'kotlin-android'
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
```

Comment:
`id "org.jetbrains.kotlin.android"` or use the `alias(versioncataloggoeshere)`

Reply:
done!